test/diffeqoperator.jl

```diff
@@ -1,39 +0,0 @@
-using SciMLBase
-using LinearAlgebra
-
-@testset "DiffEqOperator" begin
```

Comment:
should these tests be updated to use the new operators rather than deleted?

Reply:
No, SciMLOperators.jl already has tests and they are much more extensive than this, superset it already IIRC. So I am not worried about copying these because I believe the first version already did that.